zerver/views/webhooks/pingdom.py

```diff
@@ -0,0 +1,65 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+from zerver.models import get_client
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+import ujson
+
+
+PINGDOM_SUBJECT_TEMPLATE = 'Your Pingdom {name} descries something important.'
```

Comment:
While entertaining, this subject seems fairly verbose; given that the fact this is a Pingdom alert will likely be communicated by which bot is sending the message, it probably makes sense to have the subject just be "{name} status".